IBPSA/Airflow/Multizone/BaseClasses/powerLawFixedM.mo

```diff
@@ -22,13 +22,14 @@ protected
   Real pi2 "Square of normalized pressure";
 algorithm
  if (dp >= dp_turbulent) then
-   V_flow :=C *dp^m;
+   V_flow :=C * (if (m < 0.500000000001 and m > 0.499999999999) then sqrt(dp) else dp^m);
  elseif (dp <= -dp_turbulent) then
```

Comment:
I propose to use the isEqual() function:

V_flow :=C * (if Modelica.Math.isEqual(m,0.5,1e-12) then sqrt(dp) else dp^m);